proposals/target-typed-new.md

```diff
@@ -46,54 +51,51 @@ The type of the expression would be inferred from the target-type which would be
 with the following exceptions:
 
 - **Enum types:** not all enum types contain the constant zero, so it should be desirable to use the explicit enum member.
-- **Delegate types:** if the type is inferrable, an anonymous function can already be used.
 - **Interface types:** this is a niche feature and it should be preferable to explicitly mention the type.
-- **Tuple types:** tuple literals should be used for such expressions.
 - **Array types:** arrays need a special syntax to provide the length.
+- **Struct default constructor**: this rules out all primitive types and most value types. If you wanted to use the default value of such types you could write `default` instead.
 
 All the other types that are not permitted in the *object_creation_expression* are excluded as well, for instance, pointer types.
 
-The default constructor for a value type (unless appeared explicitly in metadata) is excluded, which has the effect of excluding the primitive types and the default constructor for most value types. If you wanted to use the default value of such types you could write `default`.
-
-> **Open Issue:** What other types should be excluded?
+> **Open Issue:** should we allow delegates and tuples as the target-type?
```

Comment:
It's probably good to wait for the formal notes (@agocke should make them available next week), but here are the decisions I remember:
- tuple type: allow (many pros/cons, including discussion of `new(int, ..., int)` constraint and `(int, ..., int)` constructor for long tuple. Useful for folks migrating code between tuple and types)
- delegate type: somehow we missed that (didn't discuss)
- `throw new(...)`: disallow (guides users to heavily rely on `Exception` as opposed to dedicated types. That's undesirable)
- usage in binary operators: allow (that seems useful, unlike `i + default` and such)

Reply:
@alrz For the delegate type question, was there an example where that would be useful?

Reply:
no, I couldn't think of any. method groups and anonymous funcs are already coverted through an implicit conversion.